digits/config/caffe_option.py

```diff
@@ -192,8 +192,25 @@ def get_version(executable):
             # XXX: guess and let the user figure out errors later
             return parse_version(0,11,0)
         elif platform.system() == 'Windows':
-            # XXX: guess and let the user figure out errors later
-            return parse_version(0,11,0)
+            p = subprocess.Popen([executable,'-version'],
+                    stdout = subprocess.PIPE,
+                    stderr = subprocess.PIPE)
+            if p.wait():
+                raise config_option.BadValue(p.stderr.read().strip())
+            else:
+                pattern = 'version'
+                parsed_version = parse_version(0,11,0)
+                for line in p.stdout:
+                    if pattern in line:
+                        try:
+                            # if CAFFE for Windows is built with properly formatted version string
+                            parsed_version = parse_version(line[line.find(pattern)+len(pattern):].rstrip())
+                        except:
```

Comment:
Please don't ever use a bare `except`/`pass`. If you're expecting a particular type of error, catch it like:
```py
except ValueError:
  pass
```